app/src/main/res/layout/audioplayer_fragment.xml

```diff
@@ -51,6 +51,32 @@
             app:tint="?android:attr/windowBackground"
             android:importantForAccessibility="no"/>
 
+    <androidx.cardview.widget.CardView
+        android:id="@+id/cardViewSeek"
+        android:alpha="0"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignBottom="@+id/pager"
+        android:layout_centerHorizontal="true"
+        android:layout_marginBottom="12dp"
+        app:cardCornerRadius="8dp"
+        tools:alpha="1">
+
+        <TextView
+            android:id="@+id/txtvSeek"
+            android:gravity="center"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="?attr/seek_background"
```

Comment:
Could the background also be applied to the CardView directly? I think old Android versions sometimes display a padding around card contents - in that case, the card would have a white border.